lms/static/sass/course/_info.scss

```diff
@@ -229,5 +229,13 @@ div.info-wrapper {
         }
       }
     }
+
+    @media print {
```

Comment:
I don't like this remediation. Change the print rule to make it print nicely, not to remove the information.